### PR TITLE
refactor: execution to run

### PR DESCRIPTION
**Pull request by akhileshh**

### CODE REVIEW

File: zetta_utils/cli/main.py [hunk 1]
```diff
@@ -76,12 +83,20 @@ def validate_py_path(ctx, param, value):  # pylint: disable=unused-argument
     callback=validate_py_path,
     help="Specify additional imports. Must end with `.py`.",
 )
+@click.option(
+    "--no_heartbeat",
```
Comment (supersergiy) [inline zetta_utils/cli/main.py:87]: Not super sure but it might generate a `--no_heartbeat/--no_no_heartbeat`

File: zetta_utils/cli/main.py [hunk 2]
```diff
@@ -103,10 +118,13 @@ def run(
     if parallel_builder:
         zetta_utils.builder.PARALLEL_BUILD_ALLOWED = True
 
-    result = zetta_utils.builder.build(spec, parallel=parallel_builder)
-    logger.debug(f"Outcome: {pprint.pformat(result, indent=4)}")
-    if pdb:
-        breakpoint()  # pylint: disable=forgotten-debug-statement # pragma: no cover
+    if no_heartbeat:
+        _heartbeat_interval = -1
```
Comment (supersergiy) [inline zetta_utils/cli/main.py:122]: Maybe `None` is cleaner?

**Review by supersergiy — COMMENTED** [review]
Looks solid so far!

File: zetta_utils/cli/main.py (hunk 1 above)
Comment (akhileshh) [reply zetta_utils/cli/main.py:87]: That is only when a `/` is added by the user - [boolean-flags](https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags).
Should we do a `@click.option('--heartbeat/--no-heartbeat', default=True)` then? Feels more intuitive.

**Review by supersergiy — APPROVED** [review]
Looks excellent!

**Pull request merged**